src/server.c

```diff
@@ -7001,6 +7001,16 @@ int main(int argc, char **argv) {
         redis_check_rdb_main(argc,argv,NULL);
     else if (strstr(exec_name,"valkey-check-aof") != NULL)
         redis_check_aof_main(argc,argv);
+    
+    /* If enable USE_REDIS_SYMLINKS, valkey may install symlinks like 
+     * redis-server -> valkey-server, redis-check-rdb -> valkey-check-rdb,
+     * redis-check-aof -> valkey-check-aof, etc. */
+#ifdef USE_REDIS_SYMLINKS
+    if (strstr(exec_name,"redis-check-rdb") != NULL)
```

Comment:
Strstr is just seaching for a string in a string. I'm thinking we can just search for "-check-rdb" so it matches both redis and valkey in the same check and we don't need the ifdef. Do you see any risk with that?

Reply:
The code should run without risk unless the user creates a symlink named `*-check-rdb` and link to `valkey-server`, that's very, very unlikely.

I thought of doing the same thing at first, but then I considered that maybe later we would remove the symlinks support. At that point, we need to change `-check-rdb` back to `valley-check-rdb`,  I chose `#ifdef` so that we could clearly remove this part of the code.

Reply:
OK